src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -32,21 +34,26 @@
     protected $deprecationMessage = null;
     protected $equivalentValues = array();
     protected $attributes = array();
+    protected $pathSeparator;
 
     /**
-     * @param string        $name   The name of the node
-     * @param NodeInterface $parent The parent of this node
+     * Constructor.
```

Comment:
Didn't we recently remove all comments like this?

Reply:
Maybe it was reintroduced by my rebase. I will remove it.